docs/Community/sig-optee/OWNERS

```diff
@@ -0,0 +1,11 @@
+chair:
+  - dthaler
```

Comment:
I'd say someone else (maybe Hernan) would be better at this one

Reply:
I've removed all the chair appointments from this pass, so that SIGs can properly self-organize :)